Assets/HoloToolkit/Input/Scripts/InputSources/EditorHandsInput.cs

```diff
@@ -326,7 +387,7 @@ private void SendHandVisibilityEvents()
             // Send event for new hands that were added
             foreach (uint newHand in newHands)
             {
-                inputManager.RaiseSourceDetected(this, newHand);
+                InputManager.Instance.RaiseSourceDetected(this, newHand);
```

Comment:
I have a personal preference here for using the `inputManager` field from the inherited BaseInputSource class instead of `InputManager.Instance` everywhere. Besides being more verbose, its also ever so slightly  less efficient to perform property access vs a field.